lib/resources/aws/aws_config_delivery_channel.rb

```diff
@@ -0,0 +1,67 @@
+class AwsConfigurationDeliveryChannel < Inspec.resource(1)
```

Comment:
I'd suggest we rename this - the name of the AWS service is Config, not Configuration